crates/sui/src/client_commands.rs

```diff
@@ -1174,21 +1174,25 @@ impl WalletContext {
             .read_api()
             .get_objects_owned_by_address(address)
             .await?;
-
+        let o_ref_clone = object_refs.clone();
         // TODO: We should ideally fetch the objects from local cache
-        // TODO: replace with multi-get
         let mut values_objects = Vec::new();
-        for oref in object_refs {
-            let response = client
-                .read_api()
-                .get_object_with_options(oref.object_id, SuiObjectDataOptions::full_content())
-                .await?;
+        let oref_ids: Vec<ObjectID> = object_refs.into_iter().map(|oref| oref.object_id).collect();
```

Comment:
You should be able to get rid of the clone in 1177 if you replace `into_iter` with `iter`

Reply:
I think because the iter value gets called twice, I am not able to borrow from the value again.